classes/order/OrderState.php

```diff
@@ -162,4 +162,24 @@ public function isRemovable()
     {
         return !($this->unremovable);
     }
+
+    /**
+     * @param string $name
+     * @param int $idLang
+     * @param int|null $excludeIdOrderState ID of the order state excluded for the search
+     *
+     * @return int
+     */
+    public static function countByLocalizedName(string $name, int $idLang, ?int $excludeIdOrderState): int
```

Comment:
Does it make sense to use count when normally this can only return 0 or 1?

This looks like an "exists" assertion.

Also, I'm worried about executing as many requests as languages you have installed.

Reply:
Fixed the naming. Executing is stopped when a localized name exist.